sdk/src/contracts/collections/features/getIssueBufferSizeForAddressAndToken.ts

```diff
@@ -0,0 +1,93 @@
+import { BigNumber } from 'ethers';
+import { asAddress, isSameAddress, ZERO_ADDRESS } from '../..';
+import { CollectionContract } from '../collections';
+import { SdkError, SdkErrorCode } from '../errors';
+import type { TokenId, WriteOverrides } from '../types';
+import { FeatureFunctionsMap } from './feature-functions.gen';
+import { asCallableClass, ContractFunction } from './features';
+
+const GetIssueBufferSizeForAddressAndTokenFunctions = {
+  sft: 'getIssueBufferSizeForAddressAndToken(address,uint256)[uint256]',
+} as const;
+
+const GetIssueBufferSizeForAddressAndTokenPartitions = {
+  sft: [...FeatureFunctionsMap[GetIssueBufferSizeForAddressAndTokenFunctions.sft].drop],
+};
+type GetIssueBufferSizeForAddressAndTokenPartitions = typeof GetIssueBufferSizeForAddressAndTokenPartitions;
+
+const GetIssueBufferSizeForAddressAndTokenInterfaces = Object.values(
+  GetIssueBufferSizeForAddressAndTokenPartitions,
+).flat();
+type GetIssueBufferSizeForAddressAndTokenInterfaces = (typeof GetIssueBufferSizeForAddressAndTokenInterfaces)[number];
+
+export type GetIssueBufferSizeForAddressAndTokenArgs = {
+  owner: string;
+  tokenId: TokenId;
+};
+
+export type GetIssueBufferSizeForAddressAndTokenCallArgs = [
+  args: GetIssueBufferSizeForAddressAndTokenArgs,
+  overrides?: WriteOverrides,
+];
+export type GetIssueBufferSizeForAddressAndTokenResponse = IssueBufferSize;
+
+export type IssueBufferSize = BigNumber;
+
+export class GetIssueBufferSizeForAddressAndToken extends ContractFunction<
+  GetIssueBufferSizeForAddressAndTokenInterfaces,
+  GetIssueBufferSizeForAddressAndTokenPartitions,
+  GetIssueBufferSizeForAddressAndTokenCallArgs,
+  GetIssueBufferSizeForAddressAndTokenResponse
+> {
+  readonly functionName = 'getIssueBufferSizeForAddressAndToken';
+
+  constructor(base: CollectionContract) {
+    super(
+      base,
+      GetIssueBufferSizeForAddressAndTokenInterfaces,
+      GetIssueBufferSizeForAddressAndTokenPartitions,
+      GetIssueBufferSizeForAddressAndTokenFunctions,
+    );
+  }
+
+  execute(
+    ...args: GetIssueBufferSizeForAddressAndTokenCallArgs
+  ): Promise<GetIssueBufferSizeForAddressAndTokenResponse> {
+    return this.getIssueBufferSizeForAddressAndToken(...args);
+  }
+
+  async getIssueBufferSizeForAddressAndToken(
+    args: GetIssueBufferSizeForAddressAndTokenArgs,
+    overrides: WriteOverrides = {},
+  ): Promise<IssueBufferSize> {
+    switch (this.base.tokenStandard) {
+      case 'ERC1155':
+        return await this.getIssueBufferSizeForAddressAndTokenERC1155(args, overrides);
+    }
+    this.notSupported();
+  }
+
+  protected async getIssueBufferSizeForAddressAndTokenERC1155(
+    { owner, tokenId }: GetIssueBufferSizeForAddressAndTokenArgs,
+    overrides: WriteOverrides = {},
```

Comment:
Its a get function, so we can go with `CallOverrides`:
```suggestion
    overrides: CallOverrides = {},
```